src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -72,6 +72,7 @@ def retrieveJobSplitParams(wmWorkload, task):
     if not task:
         return {"files_per_job": 5}
     else:
+        task.setSplittingParameters(allowCreationFailure=allowCreationFailure) 
```

Comment:
This is going to set a new attribute for this task. I suggest we do not set anything, but simply propagate this component parameter all the way to the job splitting algorithm.

Reply:
Hm, having only a component parameter means this can only be changed for all jobs running in one instance. We wouldn't be able to just use this for PromtpReco for instance.

Currently we could have the component parameter set to True and then override in the spec the job splitter parameter, right?

Reply:
Ahh, I made @amaltaro requested change before reading @hufnagel comment. Should I change it back to the original approach (setting a new attribute for the task)?